### PR TITLE
Hopefully fixes Icon not staying blue if no cookies are found.

### PR DESCRIPTION
**Pull request by kennethtran93**
...If you're wondering, both `cookieLength = undefined` and `cookieLength = 0` both return `false` in conditionals.

Signed-off-by: Kenneth Tran <kennethtran93@users.noreply.github.com>

### CODE REVIEW

**Comment by kennethtran93** [comment]
[Cookie-AutoDelete_Dev_20200615_221457_Chrome.zip](https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/files/4783259/Cookie-AutoDelete_Dev_20200615_221457_Chrome.zip)
[Cookie-AutoDelete_Dev_20200615_221457_Firefox.zip](https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/files/4783260/Cookie-AutoDelete_Dev_20200615_221457_Firefox.zip)

**Comment by kennethtran93** [comment]
LOL the coverage increased because I fixed a bug...

File: src/services/BrowserActionService.ts [hunk 1]
```diff
@@ -175,7 +175,7 @@ export const checkIfProtected = async (
           break;
       }
     } else {
-      if (cookieLength && cookieLength === 0) {
+      if (cookieLength !== undefined && cookieLength === 0) {
```
Comment (ke-d) [inline src/services/BrowserActionService.ts:178]: I don't understand. Isn't doing a truthy check or cookieLength good enough? As it is checking for both undefined and null?

Comment (kennethtran93) [reply src/services/BrowserActionService.ts:178]: I had left a note in the PR comment but I'll re-iterate here:

when cookieLength is undefined OR defined as number 0, both return false in conditionals.

**Review by ke-d — APPROVED** [review]

File: src/services/BrowserActionService.ts (hunk 1 above)
Comment (kennethtran93) [reply src/services/BrowserActionService.ts:178]: oh wait...did I just correct myself there...

**Pull request merged**